src/UI/Card.jsx

```diff
@@ -1,5 +1,15 @@
-const Card = () => {
-  return <></>
+import style from './Card.module.scss'
+
+const Card = ({title, description, icon, modifier}) => {
+  return <div className={`${style.card} ${style[`card--${modifier}`]}`}>
```

Comment:
It's up to you, but you could handle the "modifier" logic with a dictionary of modifiers instead of just a plain string.